schemas/documents/shared/projectPrefinanceTimelineStatus.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'projectPrefinanceTimelineStatus',
```

Comment:
I made this as a document so a same status can be re-used across different prefinance projects